script/discourse

```diff
@@ -56,210 +61,201 @@ class DiscourseCLI < Thor
         do_remap(from, to, options[:regex])
       end
     else
+      puts "", "Remapping tables on #{RailsMultisite::ConnectionManagement.current_db}...", ""
       do_remap(from, to, options[:regex])
     end
   end
 
   desc "backup", "Backup a discourse forum"
+  option :s3_uploads, type: :boolean, default: false, desc: "Include s3 uploads in the backup"
   def backup(filename = nil)
     load_rails
-    require "backup_restore"
-    require "backup_restore/backuper"
 
     store = BackupRestore::BackupStore.create
 
     if filename
-      destination_directory = File.dirname(filename).sub(/^\.$/, '')
+      destination_directory = File.dirname(filename).sub(/^\.$/, "")
 
       if destination_directory.present? && store.remote?
         puts "Only local backup storage supports paths."
         exit(1)
       end
 
-      filename_without_extension = File.basename(filename).sub(/\.(sql\.)?(tar\.gz|t?gz)$/i, '')
+      filename_without_extension = File.basename(filename).sub(/\.(sql\.)?(tar\.gz|t?gz)$/i, "")
     end
 
-    puts "Starting backup..."
-    backuper = BackupRestore::Backuper.new(Discourse.system_user.id, filename: filename_without_extension)
-    backup_filename = backuper.run
-    exit(1) unless backuper.success
+    old_include_s3_uploads_in_backup = SiteSetting.include_s3_uploads_in_backups
```

Comment:
This is **part 2** of the change introduced in this PR. Saving the old value of that site setting.